src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -92,8 +95,18 @@ extends AbstractMap[Key, Value]
     deleted = 0
   }
 
+  /** Return the index of the first slot in the hash table (in probe order)
+    * that either is empty, or is or was last occupied by the given key.
+    */
   private[this] def findIndex(key: Key) : Int = findIndex(key, hashOf(key))
 
+  /** Return the index of the first slot in the hash table (in probe order)
+    * that either is empty, or is or was last occupied by the given key.
```

Comment:
No, "is or was" is correct and crucial to understanding what the routine does, which is to return the entry which might have `Some(value)` or might have `None` for the value meaning that it was deleted.  Hence "is or was".  Maybe it needs to be expanded to make that logic clear.